README.md

```diff
@@ -58,3 +58,54 @@ HUBSPOT_CAMPAIGN_ID=<campaign id>
 ```
 
 If you do not have access to 1Password, please reach out to one of the maintainers of the project.
+
```

Comment:
I finally realized some of these are in the training style guide. Should I just add them there and mention here that it can be found there?

Reply:
No. These two documents serve very different purposes, so duplicating some of the content is fine.